customer-config/general-ledger-export/gle.expense-rules.v1.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$id": "gle.expense-rules.v1.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Expense Accounting Rules",
+  "description": "List of Expense rules for general ledger accounting and export",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "type": "string"
+      },
+      "config": {
+        "type": "object",
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "REASON_CODE",
+              "PAID_IN",
```

Comment:
Do we need PAID_IN here? Cause this is for expenses only, which is PAID_OUT. 
And btw, do we also need PAID_OUT? Since this is for expense only, we only need to specify a reason code, otherwise it's DEFAULT, right?